content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -24,6 +24,8 @@ operation:
   > repository, only [DVC-files](/doc/user-guide/dvc-file-format) that are
   > needed to download or reproduce them.
 
+- `.dvc/plots`: Directory for `dvc plots` templates.
```

Comment:
```suggestion
- `.dvc/plots`: Directory for [Plot templates](http://localhost:8000/doc/command-reference/plot#plot-templates).
```
Notice that instance of "plot" is singular. I guess that makes sense, but did you check all instances including with upper case? Will have to decide which ones stay plural in regular text on a case-by-case basis.

Reply:
@jorgeorpinel Thanks! Fixed.

> but did you check all instances including with upper case?

Yeah, looked through them, but `plots` is a command and in text we still use `Plot` to describe a plot not multiple plots, so I think that is alright.